### PR TITLE
Dont open the supply modal when toggling collateral

### PR DESCRIPTION
**Pull request by coreyar**
This PR checks if the event target is the collateral toggle when clicking the table row. If the event target has the ariaLabel equal to the switch we don't open the modal

### CODE REVIEW

**Review by baomastr — APPROVED** [review]

File: src/components/v2/Toggle/index.tsx [hunk 1]
```diff
@@ -10,14 +10,14 @@ export interface IToggleProps {
...
-const label = { inputProps: { 'aria-label': 'Switch' } };
+export const switchAriaLabel = 'Switch';
+const label = { inputProps: { 'aria-label': switchAriaLabel } };
 
 export const Toggle = ({ onChange, value, className }: IToggleProps) => {
   const styles = useStyles();
   return (
     <Switch
-      className={className}
+      className={`${className} toggle`}
```
Comment (therealemjy) [inline src/components/v2/Toggle/index.tsx]: Just curious, what is the `toggle` class used for here?

File: src/pages/Dashboard/Markets/SupplyMarket/index.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  it('click row opens modal', async () => {
+    const { getByText } = renderComponent(<SupplyMarket />);
+    const rowElement = getByText(assetData[2].symbol);
+    fireEvent.click(rowElement);
+    const connectButton = getByText(en.supplyWithdraw.connectWalletToSupply);
+    expect(connectButton).toBeTruthy();
+    await waitFor(() => expect(enterMarkets).toHaveBeenCalledTimes(0));
+  });
+
+  it('clicking toggl only toggles collateral', async () => {
```
Comment (therealemjy) [inline src/pages/Dashboard/Markets/SupplyMarket/index.spec.tsx]: small typo:
```suggestion
  it('clicking toggle only toggles collateral', async () => {
```

File: src/pages/Dashboard/Markets/SupplyMarket/index.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+describe('components/SupplyMarket', () => {
+  beforeEach(() => {
+    (useUserMarketInfo as jest.Mock).mockImplementation(() => ({
+      assets: assetData,
+      userTotalBorrowLimit: new BigNumber('111'),
+      userTotalBorrowBalance: new BigNumber('91'),
+    }));
+  });
+
+  it('click row opens modal', async () => {
```
Comment (therealemjy) [inline src/pages/Dashboard/Markets/SupplyMarket/index.spec.tsx]:
```suggestion
  it('clicking row opens modal', async () => {
```

File: src/pages/Dashboard/Markets/SupplyMarket/index.tsx [hunk 4]
```diff
@@ -165,14 +167,15 @@ const SupplyMarket: React.FC = () => {
...
-        exitMarkets({ vtokenAddress: asset.vtokenAddress, account });
+        exitMarkets({ vtokenAddress: asset.vtokenAddress, account: account?.address });
       } catch (error) {
         throw new ToastError(
           t('markets.errors.collateralDisableError.title'),
           t('markets.errors.collateralDisableError.description', { assetName: asset.symbol }),
         );
       }
     } else {
+      console.log('hhhhhhh');
```
Comment (therealemjy) [inline src/pages/Dashboard/Markets/SupplyMarket/index.tsx]: Should be removed

File: src/components/v2/Toggle/index.tsx (hunk 1 above)
Comment (coreyar) [reply src/components/v2/Toggle/index.tsx]: I don't remember why I added it but I didn't end up using it so it can be removed

**Review by therealemjy — APPROVED** [review]

**Pull request merged**